### DISCUSSION TITLE
Cannot update raspotify

### DISCUSSION THREAD

**Discussion by blokpardi** [dtcooper/raspotify#588] [Q&A] [answered]
Whenever I try to update raspotify I get the following:  

The following packages have unmet dependencies:
 raspotify : Depends: libc6 (>= 2.31) but 2.28-10+rpt2+rpi1+deb10u1 is to be installed
             Depends: libasound2 (>= 1.2.4)
             Depends: alsa-utils (>= 1.2.4) but 1.1.8-2+rpt1 is to be installed
             Depends: libpulse0 (>= 14.2) but 12.2-4+deb10u1+rpt3 is to be installed
             Depends: systemd (>= 247.3) but 241-7~deb10u8+rpi1 is to be installed
             Depends: init-system-helpers (>= 1.60) but 1.56+nmu1 is to be installed

Raspotify used to work (worked great for the last 18 months), but suddenly died and I can't get it back. This is an RPi4, and I've done a full package update. But the packages listed are all showing as on the latest version which is lower than what is shown in the dependencies. Anyone else run into this?

**Comment by JasonLG1979** [comment, marked as answer]
As stated in the [README](https://github.com/dtcooper/raspotify#readme) Raspotify only supports Debian Stable. If you installed Raspberry Pi OS 18 months ago and haven't upgraded to Stable since it's release on 2021-10-30 you are now running old-stable.

**Reply by blokpardi** [reply]
That was it. I did an update/upgrade, but I didn't realize it was just updating the previous version. Once I knew what to look for I did an OS update to the current stable version and that fixed it. Thanks for the help.

**Reply by JasonLG1979** [reply]
Good deal.

You should update your system WAY more often then every 18 months. I generally run `sudo apt update && sudo apt dist-upgrade` once a week or so to stay current and upgrade distro versions as soon as they come out.

**Reply by JasonLG1979** [reply]
Raspotify is not stable (hense the 0.x version number) so to make sure you're running the latest Raspotify version you'll need to run `dist-upgrade` because dependencies and configs may change and in the case of configs you may need to manually edit the config to resolve any conflicts or changes.

**Comment by JasonLG1979** [comment]
Duplicate of #565, #564, and #511.
